### ISSUE TITLE
[loss] add focal loss support

### ISSUE THREAD

**Issue by tijyojwad** [clara-parabricks/VariantWorks#130]
focal loss has shown to improve convergence on skewed distribution datasets. it should be applicable broadly to almost all use cases in genomics. we should add focal loss supported cross entropy to VW loss functions list. https://arxiv.org/abs/1708.02002
